### ISSUE TITLE
Arguments for Training Command

### ISSUE THREAD

**Issue by Ding-Liu** [jt827859032/DRRN-pytorch#3]
Hello! May I know the value of arguments that are used in the training command: 
main.py [-h] [--batchSize BATCHSIZE] [--nEpochs NEPOCHS] [--lr LR]
               [--step STEP] [--cuda] [--resume RESUME]
               [--start-epoch START_EPOCH] [--clip CLIP] [--threads THREADS]
               [--momentum MOMENTUM] [--weight-decay WEIGHT_DECAY]
               [--pretrained PRETRAINED]

to get the PSNRs in the following table.
| Scale   | DRRN_B1U25 Paper | DRRN_B1U25 PyTorch|
| -------:| ----------------:| -----------------:|
| x2      | 37.74            | 37.69             |
| x3      | 34.03            | 34.02             |
| x4      | 31.68            | 31.70             |

For example, the values of 'nEpochs', 'step', 'batchSize' and so on...
Many thanks in advance!

**Comment by jt827859032** [comment]
All parameters are set as default values. While during the training phase, I manually adjusted the `step` value to facilitate the training phase. :）

**Comment by Ding-Liu** [comment]
Thanks so much for your reply! 
I used your code trying to reproduce the result, but my model is ~0.2dB lower than your model in terms of performance.
1) Do you mean ```step``` is set as 5 when you did your training?
2) It seems the training datafile has the name: ```data/train_291_32_x234.h5```, which is slightly different from the name generated by the given Matlab code ```data/train_291_31_x234.h5```. Does it means the ```size_input``` is 32 rather than 31 when your training data is generated?

**Comment by jt827859032** [comment]
Hi, @Ding-Liu 
1. The initial step is 5 and during training in my experiment. While this value will be enlarged during training. I manually did this in the training phase and you can also change the adjustable learning rate policy [here](https://github.com/jt827859032/DRRN-pytorch/blob/master/main.py#L84:5). 
2. Yeah, the h5 file contains the pathes with the size of 32 x 32.
